src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -172,12 +173,18 @@ private void linkToCaseWithoutRevoking(
     }
 
     private List<Claim> getAllCasesBy(User user, ImmutableMap<String, String> searchString) {
-        List<CaseDetails> validCases = searchAll(user, searchString)
-            .stream()
-            .filter(caseDetails -> !isCreatedState(caseDetails))
+        return extractClaims(searchAll(user, searchString));
+    }
+
+    private List<Claim> getAllIssuedCasesBy(User user, ImmutableMap<String, String> searchString) {
+        List<CaseDetails> issuedCases = asStream(searchAll(user, searchString))
+            .filter(isCaseIssued())
             .collect(Collectors.toList());
+        return extractClaims(issuedCases);
+    }
 
-        return extractClaims(validCases);
+    private Predicate<CaseDetails> isCaseIssued() {
```

Comment:
`private Predicate<CaseDetails> isCaseIssued = caseDetails -> caseDetails.getState() == CREATE.getValue();` this is the correct way of declaring.

Reply:
done